packages/verified-fetch/src/utils/responses.ts

```diff
@@ -40,7 +43,20 @@ export function okResponse (url: string, body?: BodyInit | null, init?: Response
   return response
 }
 
-export function notSupportedResponse (url: string, body?: BodyInit | null, init?: ResponseInit): Response {
+export function internalServerErrorResponse (url: string, body?: SupportedBodyTypes, init?: ResponseInit): Response {
```

Comment:
not sure if we want this to be `badInternalResponse` or other, so i went verbose

Reply:
I think call it what it is, 500 is `Internal Server Error` so that's what we should call it.

By that logic `notSupportedResponse` should also be `notImplementedResponse`, but whatevs.